src/Component/Main/TodoItem.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import '../../styles/index.css';
+
+export const TodoItem = (props) => {
+  const [newTitle, setNewTitle] = useState('');
+  const [editStatus, setEditStatus] = useState(false);
+
+  const {
+    completeTodo,
+    itemId,
+    itemTitle,
+    deleteTodo,
+    itemCompleted,
+    onAddNewTitle,
+  } = props;
+
+  const soldCheckbox = () => {
+    completeTodo(itemId);
+  };
+
+  const onBlur = () => {
+    if (editStatus && (newTitle.length > 0)) {
+      const obj = {
+        id: itemId,
+        title: newTitle,
+      };
+
+      onAddNewTitle(obj);
+
+      setEditStatus(false);
+      setNewTitle('');
+    }
+  };
+
+  const onKeyPush = (keyPush) => {
+    if (keyPush === 'Enter') {
+      onBlur();
+    } else if (keyPush === 'Escape') {
+      onAddNewTitle(itemTitle);
+
+      setEditStatus(false);
+      setNewTitle('');
+    }
+  };
+
+  return (
+    <>
+      <li className={classNames({
+        completed: itemCompleted,
+        editing: editStatus,
+      })}
+      >
+        <div className="view">
+          <input
+            type="checkbox"
+            className="toggle"
+            checked={itemCompleted}
+            onChange={() => soldCheckbox()}
```

Comment:
no need to create one more func
```suggestion
            onChange={() => completeTodo(itemId)}
```